airbytehq/airbyte
# Bump Airbyte version from 0.39.24-alpha to 0.39.25-alpha

*IMPORTANT: Only merge if the platform build is passing!*

Changelog:

40b43adf73 Remove skiptests option (#14100)
01a366ca57 🐛 Destination Redshift: Improved discovery for redshift-destination not SUPER streams (#13690)
4c6d518154 fix stream name in stream transformation update (#14044)
e706e02f0d fix(jsonSchemas): raise error when items property not provided (#14018)
c8d58e8f7a Source PayPal Transaction: Update Transaction Schema (#13682)
6ca0e53469 12708: Add an option to use encryption with staging in Redshift Desti… (#14013)
4b340c4b04 Update webflow.md (#14083)
41e88a8dda Source Hubspot: extend error logging (#14054)
273fcaa824 fix: format VaultSecretPersistenceTest.java (#14110)
21b43f56cc 🐛 Source Hubspot: remove `AirbyteSentry` dependency (#14102)
007e7b5258 secret-persistence: Hashicorp Vault Secret Store (#13616)
f3c54b5c67 Add StatePersistence object (#13900)
7f11c91921 Emit the state to remove in the airbyte empty source (#13725)

Instructions:
- *SQUASH MERGE* this PR - this is necessary to ensure the automated Create Release action is triggered.
- Double check that the Create Release action was triggered and ran successfully on the commit to master (this should only take a few seconds).
- If the Create Release action failed due to a transient issue, retry the action. If it failed due to a non-transient issue, you will need to manually create a release by following these steps:
 1. Pull most recent version of master
 2. Run ./tools/bin/tag_version.sh
 3. Create a GitHub release with the changelog
